pkg/kube_events_manager/types/types.go

```diff
@@ -80,17 +74,29 @@ func (o ObjectAndFilterResult) MarshalJSON() ([]byte, error) {
 
 func (o *ObjectAndFilterResult) RemoveFullObject() {
 	o.Object = nil
-	o.ObjectBytes = 0
+	o.ObjectSize = 0
 	o.Metadata.RemoveObject = true
 }
 
 type ObjectAndFilterResults map[string]*ObjectAndFilterResult
 
-func (a ObjectAndFilterResults) Bytes() (size int64) {
+func (a ObjectAndFilterResults) Bytes() (size int) {
 	for _, o := range a {
-		size += int64(len(o.FilterResult))
-		size += o.ObjectBytes
+		if newSize, overflowed := overflow.Add(size, o.FilterResultSize); overflowed {
+			size = mathutil.MaxInt
```

Comment:
I don't get it, why use int instead of int64?

Reply:
Reverted to int64.